webapi/src/main/scala/org/knora/webapi/routing/Authenticator.scala

```diff
@@ -869,12 +865,7 @@ final case class JwtServiceLive(private val config: AppConfig, stringFormatter:
         ).contains(false)
 
         if (!missingRequiredContent) {
-          Try(
-            stringFormatter.validateAndEscapeIri(
-              claim.subject.get,
-              throw BadRequestException("Invalid user IRI in JWT")
-            )
-          ).fold(_ => None, _ => Some(header, claim))
+          claim.subject.flatMap(iri => stringFormatter.validateAndEscapeIri(iri).toOption.map(_ => (header, claim)))
```

Comment:
previously this was done in `Try(...).fold(_ => None, _ => Some(header, claim)` so I don't think any of the errors made it out. But probably this method should be changed to return `Validation` instead of `Option`, that would make it clearer. Or am I messing up, here?

Reply:
No, you are right. I have missed that the `.fold` logic is actually mapping to a `None` in case of the `NoSuchElementException`, i.e. the behaviour did not change.